packages/gatsby-source-wordpress/src/steps/auth-handling.ts

```diff
@@ -0,0 +1,28 @@
+import type { GatsbyNodeApiHelpers } from "~/utils/gatsby-types"
+import type { Step } from "./../utils/run-steps"
+import type { IPluginOptions } from "~/models/gatsby-api"
+
+let storedAuthSettings: IPluginOptions["auth"] | undefined
```

Comment:
not greatest to store as module variable, but I lack context on this plugin internals, so if there is a better way - happy to adjust

Reply:
I think this is ok. The plugins internals aren't great 😜 wish I could undo the way it manages state so this is preferrable imo.